src/manifest.rs

```diff
@@ -136,9 +137,15 @@ impl BuildInfo {
         if let Some(workdir) = self.workdir {
             cmd.current_dir(workdir);
         }
-        cmd.args(["-c", &self.cmd])
+        let status = cmd
+            .args(["-c", &self.cmd])
             .status()
-            .context("failed to build component")?;
-        Ok(())
+            .context("failed to build test component")?;
+
+        if status.success() {
+            return Ok(());
+        }
+
+        Err(anyhow!("failed to build test component. {}", status))
```

Comment:
Should this output the stderr of the command?

Reply:
IIUC the output of the stderr is already displayed on the console